binding/binding.go

```diff
@@ -43,17 +43,17 @@ type BindingUri interface {
 	BindUri(map[string][]string, interface{}) error
 }
 
-// StructValidator is the minimal interface which needs to be implemented in
+// ValidatorImp is the minimal interface which needs to be implemented in
 // order for it to be used as the validator engine for ensuring the correctness
 // of the request. Gin provides a default implementation for this using
 // https://github.com/go-playground/validator/tree/v8.18.2.
-type StructValidator interface {
-	// ValidateStruct can receive any kind of type and it should never panic, even if the configuration is not right.
+type ValidatorImp interface {
```

Comment:
Please use the Golang [interface naming conversion](https://golang.org/doc/effective_go.html#interface-names).